packages/rsocket-ws-gateway/src/Gateway.ts

```diff
@@ -10,10 +10,14 @@ export class Gateway implements GatewayInterface {
   private started = false;
   private server;
   private transport;
+  private requestResponse;
+  private requestStream;
   constructor(opts: GatewayOptions) {
     const { port } = opts;
     this.port = port || 3000;
     this.transport = new RSocketWebSocketServer({ port: this.port });
+    this.requestResponse = opts.requestResponse;
+    this.requestStream = opts.requestStream;
```

Comment:
add validation:
this.requestResponse - opts.requestResponse || defaultValue --> check value is function
this.requestStream - same